src/zephyr/tool/get-projects.ts

```diff
@@ -0,0 +1,40 @@
+import type { ToolCallback } from "@modelcontextprotocol/sdk/server/mcp.js";
+import type { ApiClient } from "../common/api-client.js";
+import { MaxResultsSchema, StartAtSchema } from "../common/types.js";
+import type { ZephyrTool } from "./zephyr-tool.js";
+
+export class GetProjects implements ZephyrTool {
+  private readonly apiClient: ApiClient;
+
+  constructor(apiClient: ApiClient) {
+    this.apiClient = apiClient;
+  }
+
+  specification = {
```

Comment:
I know that in projects we have by now only get projects but I don't think we should call the tool at the same way that one endpoint, so it can cause later confusion with next ones... f.e: I do not expect having toos for create-test-case but to have test case where we could do different actions...

Reply:
The idea is that each separate class will be a different tool. It makes it easier to make a shorter scope.

For example:
* **Get Projects** and **Get Project** are different tools; therefore, they will be different classes

We can also discuss this too

Reply:
I would gather them by entity/domain, here we will end up like having an adapter per endpoint (if we translate to our backend project)